tests/data/test_sampler.py

```diff
@@ -12,34 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import numpy as np
 import os
+import unittest
 
 from paddlenlp.data import SamplerHelper
 from paddlenlp.datasets import load_dataset
-
-from common_test import CpuCommonTest
-import util
-import unittest
+from tests.common_test import CpuCommonTest
+from tests.testing_utils import assert_raises, get_tests_dir
 
 
 def cmp(x, y):
     return -1 if x < y else 1 if x > y else 0
 
 
 class TestSampler(CpuCommonTest):
-    def setUp(self):
-        self.config["path"] = "imdb"
-        self.config["splits"] = "train"
-        self.train_ds = load_dataset(**self.config)
+    @classmethod
+    def setUpClass(cls):
+        fixture_path = get_tests_dir(os.path.join("fixtures", "dummy"))
+        cls.train_ds = load_dataset("clue", "tnews", data_files=[os.path.join(fixture_path, "tnews", "train.json")])
```

Comment:
imdb is 80M of data download, which is too big for a CI that is run with every push. Changing this to a dummy dataset stored in fixtures.